tests/crashes/116979.rs

```diff
@@ -0,0 +1,14 @@
+//@ known-bug: #116979
+//@ compile-flags: -Csymbol-mangling-version=v0
+//@ needs-rustc-debug-assertions
+
+#![feature(dyn_star)]
+#![allow(incomplete_features)]
```

Comment:
Question/discussion: should we even add crashes tests for incomplete features...? Cuz like, it's known to be incomplete, right? I feel like we should just backlink the bug report to the tracking issue (and tag it along with the github subissues thing), because otherwise if someone wants to drive `dyn_star` forward a crashes test would just be friction.

cc https://rust-lang.zulipchat.com/#narrow/channel/131828-t-compiler/topic/Should.20crashes.20test.20include.20incomplete.20features.3F